lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -749,6 +749,29 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'If `true`, the agent captures log records emitted by your application.'
         },
+        :'application_logging.forwarding.log_level' => {
+          :default => 'debug',
+          :public => true,
+          :type => String,
+          :allowed_from_server => false,
+          :description => <<~DESCRIPTION
+            Sets the minimum level a log event must have to be forwarded to New Relic.
+
+            This is based on the integer values of Ruby's `Logger::Severity` constants: https://github.com/ruby/ruby/blob/master/lib/logger/severity.rb
+
+            The intention is to forward logs with the level given to the configuration, as well as any logs with a higher level of severity.
+
+            For example, setting this value to "debug" will forward all log events to New Relic. Setting this value to "error" will only forward log events with the levels "error", "fatal", and "unknown".
+
+            Valid values (ordered lowest to highest):
```

Comment:
Do we want to enforce this adherence to this allowlist? One possible use case: I typed in "warm" by mistake. Should I be told about the problem?

Reply:
There's some enforcement in the [LogEventAggregator#minimum_log_level method](https://github.com/newrelic/newrelic-ruby-agent/pull/2051/files#diff-680b1e6a8547301e66b77f383e273ef141166e5986c9a1180625081dc8e271bcR236), where values outside the valid range will be overridden with `:DEBUG`. It's a little flexible for caps/lowercase values since the configs are coerced to upcased symbols before evaluation. 

I like the idea of being told about the problem and added a log_once message to the method: 3fc3c2a1d7b8230e3b9b1b75448d4967b296a936 

If you think there are additional steps for enforcement that would benefit customers, please let me know.